pkg/kube/site/site.go

```diff
@@ -31,13 +30,11 @@ type Site struct {
 	site        *skupperv1alpha1.Site
 	name        string
 	namespace   string
-	siteId      string
-	config      *types.SiteConfigSpec
 	controller  *kube.Controller
 	bindings    *site.Bindings
 	links       map[string]*site.Link
 	errors      map[string]string
-	linkAccess  LinkAccessMap
+	linkAccess  site.RouterAccessMap
```

Comment:
Maybe rename it to routerAccess ?

Reply:
The reason it is called linkAccess is that is only controlling the access for linking.  A separate field will control management access.